server/src/termdb.filter.js

```diff
@@ -4,6 +4,9 @@ import { dtTerms } from '#shared/common.js'
 import { getSnpData } from './termdb.matrix.js'
 import { filterByItem } from './mds3.init.js'
 
+// includes term.type=date, unlike in termdb.sql.numeric.js
+const annoNumericTypes = new Set(['integer', 'float', 'date'])
```

Comment:
Maybe we can import the numeric types set defined in shared/utils/src/terms.js and check against this list to avoid duplicates. Every time we add a new term we edit the terms.js file so it is easier to remember adding it to the numeric types if it is numeric.